### PR TITLE
Imported parent components into parent home and exported parent home

### PR DESCRIPTION
**Pull request by vasqueza91**
What have you done?

Imported parent calendar, child list, and parent sidebar into parent home and exported parent home.

What files were changed?

The ParentHome.js

Why was this file changed?

This file was changed in order to import all of the parent components.

https://user-images.githubusercontent.com/82040208/139714743-0a6ae5aa-598b-42f2-8f40-c4a529d4a96a.mp4

### CODE REVIEW

**Review by cn8817 — APPROVED** [review]
Looks great!

**Review by JoeyMBrown — APPROVED** [review]
Looks good, nice job! 👍

**Comment by fernando817mm** [comment]
Awesome job! Thanks for including a preview of what it's looking like!

**Review by Jie-chelchel — APPROVED** [review]

**Review by fernando817mm — APPROVED** [review]

File: src/components/pages/ParentHome/index.js [hunk 1]
```diff
@@ -1 +1,16 @@
-export { default as ParentHome } from './ParentSidebar';
+import React from 'react';
+import ChildList from './ChildrenList';
+import ParentCalendar from './ParentCalendar';
+import ParentSidebar from './ParentSidebar';
+
+function ParentHome() {
+  return (
```
[the remark this answers is not in the document]
Comment (JoeyMBrown) [reply src/components/pages/ParentHome/index.js:7]: Great question.  This is something we originally scoped for with the current ticket - we we're lightly touching on CSS and how we were going to write it today in a meeting.  Waiting to hear back on how we should approach CSS.  For now we're considering using Less, and SCSS for styling, until we hear back it's ok for the components to render in a not so organized way :)

**Pull request merged**